src/test/java/dev/openfeature/javasdk/DeveloperExperienceTest.java

```diff
@@ -6,13 +6,23 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.*;
 
 
 @SuppressWarnings("unchecked")
 class DeveloperExperienceTest implements HookFixtures {
     transient String flagKey = "mykey";
+
+    @Test void noProviderSet() {
```

Comment:
Do you think this should be something that's written in the spec?

Reply:
I think we already say to never throw or abnormally end in any case. How to accomplish that seems like a detail. Setting a no-op provider just a nice way to do it.